pkg/security/admission/admission.go

```diff
@@ -205,8 +226,11 @@ func (c *constraint) createProvidersFromConstraints(ns string, sccs []*kapi.Secu
 		var err error
 		resolveUIDRange := requiresPreAllocatedUIDRange(constraint)
 		resolveSELinuxLevel := requiresPreAllocatedSELinuxLevel(constraint)
+		resolveFSGroup := requiresPreallocatedFSGroup(constraint)
+		resolveSupplementalGroups := requiresPreallocatedSupplementalGroups(constraint)
+		requiresNamespaceAllocations := resolveUIDRange || resolveSELinuxLevel || resolveFSGroup || resolveSupplementalGroups
 
-		if resolveUIDRange || resolveSELinuxLevel {
+		if requiresNamespaceAllocations {
```

Comment:
Below (its not in the diff), don't create a scoped `if` that doesn't scope.

```go
if namespace, err = c.getNamespace(ns, namespace); err != nil {
```
should be 
```go
namespace, err = c.getNamespace(ns, namespace)
if err != nil {
```

The former tricks me into thinking that the variables are scoped and don't leak out, the latter makes it clear that those variables are not scoped to the block.